pvlib/clearsky.py

```diff
@@ -14,7 +14,7 @@
 
 
 def ineichen(apparent_zenith, airmass_absolute, linke_turbidity,
-             altitude=0, dni_extra=1364., perez_enhancement=False):
+             altitude=0, dni_extra=1364., perez_enhancement=None):
```

Comment:
IMO I don't see broad value in having three options:

1. no Perez term
2. Perez term without air mass clipping
3. Perez term with air mass clipping

I'm OK dropping the second.